docs/get-started/quick-start-guide.md

```diff
@@ -0,0 +1,145 @@
+=======
+# Quick Start
+
+[TOC]
+
+## Step 1: Activate a virtual environment
+
+This is a Python toolkit, so you'll need to install Python (version 3.7 or greater), and setup and install a virtual environment.
+
+Yep. Even if you're not writing code—and only using the "no code" CLI part of the Toolkit—you're using Python to communicate with the Planet Labs PBC servers. It's not too tricky, but it does require a presence of mind to complete. If you need help with Python install and setting up a virtual environment, read [Getting started with venv & Planet SDK](venv-tutorial.md).
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 2: Install the Planet Earth Observation Toolkit
+
+Install with [pip](https://pip.pypa.io):
+
+```console
+$ pip install planet --user
+```
+
+The [--user](https://pip.pypa.io/en/stable/user_guide/#user-installs)
+flag is highly recommended for those new to pip.
+
+## Step 3: Get your API Key ready
+
+<<<<<<< HEAD
+Planet's APIs require an account for use. For more information on where to find your API key, or to sign up to get one, see [Get Your API Key](get-your-api-key.md).
+=======
+Planet's APIs require an account for use. For more information on where to find your API key, or get one if you don't already have one, see [Get Your API Key](get-your-api-key.md).
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 4: Check the toolkit version
+
+```console
+$ planet --version
+```
+
+<<<<<<< HEAD
+You should be on some version 2 of the toolkit. If you're not, or you need more information about versioning, early access, and upgrading, see [Early Access to Planet SDK (aka "V2")](v2_earlyaccess.md).
+=======
+You should be on some version 2 of the toolkit. If you're not, or you need to 
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 5: Sign on to your account
+
+Just as you log in when you browse to https://account.planet.com, you'll want to sign on to your account so you have access to your account and orders.
+
+## Step 6: Use the Toolkit to authenticate against the Planet servers
+
+```console
+$ planet auth init
+```
+
+You'll be prompted for the email and password you use to access [your account](https://account.planet.com).
+
+When you type in your password, you won't see any indication that the characters are being accepted. But when you hit enter, you'll know that you've succeeded because you'll see on the command line:
+
+```console
+Initialized
+```
+
+## Step 7: Creating your first order
+
+[*****TBW*****]
+
+The Planet SDK for Python allows Python developers to write software that makes
+use of the following Planet APIs:
+
+* [orders](https://developers.planet.com/docs/orders/)
+* [data](https://developers.planet.com/docs/data/) (not implemented)
```

Comment:
```suggestion
* [data](https://developers.planet.com/docs/data/) 
```

This is implemented now - have been meaning to get this in, and I think this PR is fine to fix - I realize it's not a problem with this PR.